### PR TITLE
Implement behaviour testing for cairo 1.0

### PR DESCRIPTION
**Pull request by piwonskp**
This PR provides a temporary solution to run behaviour tests for cairo 1.0. 

For more details see #1046.

### CODE REVIEW

File: tests/testnetInterface.ts [hunk 1]
```diff
@@ -102,33 +132,42 @@ export async function invoke(
...
+      status: 500,
+      steps: null,
+      threw: true,
+      return_data: null,
+      error_message: result,
+      events: [],
+    };
+  }
+  const transactionAsBuffer = await execSync(
+    `starknet get_transaction_trace --gateway_url http://devnet:5050 --feeder_gateway_url http://devnet:5050 --hash ${matches[1]}`,
```
Comment (cicr99) [inline tests/testnetInterface.ts]: Why not use `starknetCliCall` here?

File: tests/testnetInterface.ts [hunk 2]
```diff
@@ -102,33 +132,42 @@ export async function invoke(
...
+    steps: null,
+    threw: false,
+    return_data: transaction.function_invocation.result,
+    error_message: undefined,
+    events: transaction.function_invocation.events,
+  };
 }
 
 export async function ensureTestnetContactable(timeout: number): Promise<boolean> {
+  return Promise.resolve(true);
```
Comment (cicr99) [inline tests/testnetInterface.ts:170]: What are we doing with the rest of the code in this function?

File: tests/testnetInterface.ts (hunk 1 above)
Comment (piwonskp) [reply tests/testnetInterface.ts]: Good catch

File: tests/testnetInterface.ts (hunk 2 above)
Comment (piwonskp) [reply tests/testnetInterface.ts:170]: I left it for later, I think the logic of verifying server (devnet in this case) availability might be needed. I'll comment that tho to make it clear it's not used right now

File: tests/behaviour/behaviour.test.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```
Comment (rodrodros) [inline tests/behaviour/behaviour.test.ts:1]: I think there is no need to disable `eslint` in this specific file

Comment (piwonskp) [reply tests/behaviour/behaviour.test.ts:1]: There is unused import `compileCluster` that breaks  linting. Currently the function is replaced with `compileCairo1`, but I'd leave the decision on what to do with it for a PR refactoring tests. The function should be either removed or updated to work with cairo 1

Comment (piwonskp) [reply tests/behaviour/behaviour.test.ts:1]: I think it's better to leave a trace that such a function was used in the code so when fixing linting it'll have to be dealt with, not just silently ignored

**Review by rodrodros — APPROVED** [review]

**Review by cicr99 — APPROVED** [review]

**Pull request merged**